kustomize/components/clairpgupgrade/clair-pg-upgrade.job.yaml

```diff
@@ -9,84 +9,41 @@ metadata:
 spec:
   template:
     spec:
-      restartPolicy: Never
+      restartPolicy: OnFailure
       terminationGracePeriodSeconds: 180
       serviceAccountName: clair-postgres
       volumes:
         - name: clair-postgres-conf-sample
           configMap:
             name: clair-postgres-conf-sample
-        - name: postgres-data
+        - name: clair-postgres-13
           persistentVolumeClaim:
-            claimName: clair-postgres
-        - name: migration-data
-          persistentVolumeClaim:
-            claimName: clair-postgres-migration
-      initContainers:
-        - name: postgres-old
-          image: centos/postgresql-12-centos7@sha256:be8803d45d64870f8dfd018f3110af62e2e1558d64191faea461005e1bd03243
-          imagePullPolicy: IfNotPresent
-          ports:
-            - containerPort: 5432
-              protocol: TCP
-          env:
-            - name: POSTGRESQL_UPGRADE
-              value: copy
-            - name: POSTGRESQL_USER
-              value: postgres
-            - name: POSTGRESQL_DATABASE
-              value: postgres
-            - name: POSTGRESQL_PASSWORD
-              value: postgres
-            - name: POSTGRESQL_ADMIN_PASSWORD
-              value: postgres
-            - name: POSTGRESQL_MAX_CONNECTIONS
-              value: "1000"
-          volumeMounts:
-            - name: clair-postgres-conf-sample
-              mountPath: /usr/share/pgsql/postgresql.conf.sample
-              subPath: postgresql.conf.sample
-            - name: postgres-data
-              mountPath: /var/lib/pgsql/data
-            - name: migration-data
-              mountPath: /var/lib/pgsql/backup
-          resources:
-            requests:
-              cpu: 500m
-              memory: 2Gi
-          command:
-            - "/bin/bash"
-            - "-c"
-            - "cp -r /var/lib/pgsql/data /var/lib/pgsql/backup/clair && run-postgresql --help"
-
+            claimName: clair-postgres-13
```

Comment:
nit: should we name PVCs `old/new` instead of `-13` since the upgrade can be done for future pg versions too?

Reply:
if we are going to create a new PVC for every new Postgres version, we cannot use old/new. Today postgres-13 is "new", next release it'll be "old", and we cannot rename PVCs.